theme/src/resources/templates/new_request_page.hbs

```diff
@@ -107,7 +107,7 @@
 
 					// Set Nested Field id to match its label value
 
-					if (!id.startsWith('prd')) {
+					if (id.search('prd') === -1) {
```

Comment:
Can we use `indexOf` instead as that will be faster than `.search`, especially in a loop?